settings.gradle

```diff
@@ -1,2 +1,3 @@
 include ':pinwheel-android'
+include ':app'
```

Comment:
Did you intend to leave this here or was this leftover from local testing?

Reply:
Adding this back is partly the reason for deprecating Jitpack support.